common/credential-storage/src/backends/sqlite.rs

```diff
@@ -113,4 +114,58 @@ impl CoconutCredentialManager {
         .await?;
         Ok(())
     }
+
+    /// Inserts provided coin_indices_signatures into the database.
+    ///
+    /// # Arguments
+    ///
+    /// * `epoch_id`: Id of the epoch.
+    /// * `coin_indices_signatures` : The coin indices signatures for the epoch
+    pub async fn insert_coin_indices_sig(
+        &self,
+        epoch_id: String,
+        coin_indices_sig: String,
+    ) -> Result<(), sqlx::Error> {
+        sqlx::query!(
+            "INSERT INTO coin_indices_signatures(epoch_id, signatures) VALUES (?, ?)",
+            epoch_id,
+            coin_indices_sig
+        )
+        .execute(&self.connection_pool)
+        .await?;
+        Ok(())
+    }
+
+    /// Check if coin indices signatures are present for a given epoch
+    ///
+    /// # Arguments
+    ///
+    /// * `epoch_id`: Id of the epoch.
+    pub async fn is_coin_indices_sig_present(&self, epoch_id: String) -> Result<bool, sqlx::Error> {
+        sqlx::query!(
+            "SELECT epoch_id FROM coin_indices_signatures WHERE epoch_id = ?",
+            epoch_id
+        )
+        .fetch_optional(&self.connection_pool)
+        .await
+        .map(|r| r.is_some())
+    }
+
+    /// Get coin_indices_signatures of a given epoch.
+    ///
+    /// # Arguments
+    ///
+    /// * `epoch_id`: Id of the epoch.
+    pub async fn get_coin_indices_sig(
+        &self,
+        epoch_id: String,
+    ) -> Result<Option<CoinIndicesSignature>, sqlx::Error> {
+        sqlx::query_as!(
+            CoinIndicesSignature,
+            "SELECT * FROM coin_indices_signatures WHERE epoch_id = ?",
+            epoch_id
+        )
+        .fetch_optional(&self.connection_pool)
```

Comment:
Signatures are needed at spend time. 
When creating a credential, a client checks if he has the corresponding signatures. It he does, he doesn't need them right away, so let's not load them here (they aren't really small).
This can be changed to `fetch_one` though